python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -1032,6 +1089,9 @@ router bgp {{ router_bgp.as }}
    !
    address-family ipv6
 {%         for peer_group in router_bgp.address_family_ipv6.peer_groups | arista.avd.natural_sort('name') %}
+{%             if peer_group.additional_paths.receive is arista.avd.defined(true) %}
```

Comment:
missed the logic for install and install_ecmp_primary as in schema the option is provided

Reply:
Removed these keys from schema.